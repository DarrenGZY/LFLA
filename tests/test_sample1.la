function linearIndep(vector[] vectors, var n)
{
    if n==1 { return 1; }
    if n > dim(vectors[0])
        { return 0; }
    
    vecspace vs;
    for var i = 0:n
    {
        if vectors[i]@vs
            {return 0;}
        vs = vs + L(vectors[i]);
    }

    return 1;
}

function main()
{
    vector v = [1,2,3];
    vector u = [2,22,3];
    vector x[2] = { v, u};

    if linearIndep(x, 2)
    {
        print(1);
    }
}
