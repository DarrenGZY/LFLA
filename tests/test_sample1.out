Not belongs to this vectors spaces
Not belongs to this vectors spaces
1
